server/monitor.go

```diff
@@ -1341,7 +1341,11 @@ func (s *Server) updateJszVarz(js *jetStream, v *JetStreamVarz, doConfig bool) {
 	v.Stats = js.usageStats()
 	if mg := js.getMetaGroup(); mg != nil {
 		if ci := s.raftNodeToClusterInfo(mg); ci != nil {
-			v.Meta = &MetaClusterInfo{Name: ci.Name, Leader: ci.Leader, Replicas: ci.Replicas, Size: mg.ClusterSize()}
```

Comment:
Personal preference, but I would have done something like:
```
v.Meta = &MetaClusterInfo{Name: ci.Name, Leader: ci.Leader, Size: mg.ClusterSize()}
if ci.Leader == s.info.Name {
    v.Meta.Replicas = ci.Replicas
}
```
basically adding to the common setting when this is the leader.

Reply:
yeah that seems better, i will fix.